tools/deployments/open-v3-pr.ts

```diff
@@ -21,7 +21,18 @@ if (require.main === module) {
 		try {
 			// Open PR
 			execSync(
-				`gh pr create --base v3-maintenance --head v3-maintenance-${process.env.PR_NUMBER} --label "skip-pr-description-validation" --label "skip-v3-pr" --title "Backport #${process.env.PR_NUMBER} to Wrangler v3" --body "This is an automatically opened PR to backport patch changes from #${process.env.PR_NUMBER} to Wrangler v3"`
+				[
+					`gh`,
+					`pr`,
+					`create`,
+					`--base`,
+					`v3-maintenance`,
+					`--head v3-maintenance-${process.env.PR_NUMBER}`,
+					`--label "skip-pr-description-validation"`,
+					`--label "skip-v3-pr"`,
+					`--title "V3 Backport [#${process.env.PR_NUMBER}]: ${process.env.PR_TITLE?.slice(1, -1)}`,
```

Comment:
This is the key change. It adds the title of the original PR to the title of the backport PR.